src/class/CgiManager/CgiManager.cpp

```diff
@@ -14,10 +15,9 @@ CgiManager::CgiManager(const BasicHTTPRequest &basicHTTPRequest, const Path &pat
 	, _pathCGI(pathCGI)
 	, _serverName(serverName)
 	, _port(port)
+	, _byte_write(0)
+	, _byte_read(0)
 {
-	_env = Tab();
-	_argv = Tab();
-	_pipe = Pipe();
 	_basicHTTPRequest.parseRaw();
```

Comment:
why are we parsing here?
i guess this is not your code but we are already parsing the req in `ServerManager::clientRead`
i opend in issue about this so never mind it

Reply:
I have removed it anyway, you are right, it was not necessary